tests/conftest.py

```diff
@@ -41,11 +41,13 @@ def set_cwd(testdir):
     pytest_inmanta.plugin.CURDIR = os.getcwd()
 
 
-@pytest.fixture(scope="function", autouse=True)
+@pytest.fixture(scope="function")
```

Comment:
I don't think we can remove the autouse here, can we?

Reply:
I naively thought that was part of the fixes that needed to be ported, I need to think about this some more.

Reply:
In inmanta/inmanta-core##4717 the autouse was removed from the `deactive_venv` fixture because in the new implementation the `deactive_venv` fixture is called from the `clean_reset` which is an autouse fixture. So the `deactive_venv` fixture becomes autouse by indirection.

Reply:
Oh I completely missed that, I get it now